index.html

```diff
@@ -811,15 +811,28 @@ <h4>Event coordinates</h4>
-                    </ul>
-                    <p>When these conditions are met, event |e| MUST be dispatched to the capturing target of event |t|, and event |e| MUST be dispatched after the dispatch of the {{GlobalEventHandlers/lostpointercapture}} event with the same <code>pointerId</code>.</p>
-                    <div class="note">When |t| is a <code>PointerEvent</code>, its type depends on native platform convention: if |e| is a <code>contextmenu</code> event, |t| is either a {{GlobalEventHandlers/pointerdown}} or a {{GlobalEventHandlers/pointerup}} event; if |e| is a <code>click</code> or <code>auxclick</code> event, |t| is a {{GlobalEventHandlers/pointerup}} event.</div>
+                    <p>A <code>click</code>, <code>auxclick</code> or <code>contextmenu</code> event |e| MUST follow the dispatch process defined in the [[UIEVENTS]] spec except for the overrides defined in the algorithm below:</p>
+                    <ol>
+                        <li>
+                            <p>Let |u| be the user interaction event that caused the firing of |e|.</p>
+                            <div class="note">
+                                <p>Event |u| could be a non-<code>PointerEvent</code>; for example, it is a <code>KeyboardEvent</code> when a <code>click</code> event dispatch is caused by hitting the spacebar on a checkbox element.</p>
+                                <p>When |u| is a <code>PointerEvent</code>, |u| is a {{GlobalEventHandlers/pointerup}} except for a <code>contextmenu</code> event where |u| is either a {{GlobalEventHandlers/pointerdown}} or a {{GlobalEventHandlers/pointerup}} event depending on native platform convention.</p>
+                            </div>
+                        </li>
+
+                        <li>If |u| is not a <code>PointerEvent</code>, dispatch |e| following the [[UIEVENTS]] spec (the remaining steps below are skipped).</li>
+
+                        <li>
+                            <p>Define |t| as follows:</p>
+
+                            <p>If |e| is a <code>contextmenu</code> event, or |u| was dispatched while the corresponding pointer was captured, then let |t| be the target of |u|.</p>
+
+                            <p>Otherwise (|e| is a <code>click</code> or <code>auxclick</code> event for which |u| is a <code>pointerup</code> event that was dispatched uncaptured) let |t| be the nearest common inclusive ancestor of the corresponding <code>pointerdown</code> and <code>pointerup</code> targets in the DOM at the moment event |e| is being dispatched.</p>
+                        </li>
+
+                        <li>Dispatch |e| to |t| following the [[UIEVENTS]] spec.  If |u| was dispatched while the corresponding pointer was captured, |e| MUST be dispatched after the dispatch of the {{GlobalEventHandlers/lostpointercapture}} event with the same <code>pointerId</code>.</li>
```

Comment:
We already require that lostpointercapture is dispatched before any "subsequent events for the pointer" in https://w3c.github.io/pointerevents/#the-lostpointercapture-event right? It probably shouldn't be normatively required in two places.

Reply:
> We already require that lostpointercapture is dispatched before any "subsequent events for the pointer" in https://w3c.github.io/pointerevents/#the-lostpointercapture-event right? It probably shouldn't be normatively required in two places.

Made it a non-normative text.  We added it few months ago because it is an important exception that deserves attention here.